From 73637408a355fd4b2b1818e9dcf48ee1cfdd6d42 Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <jkysela@redhat.com>
Date: Wed, 3 Sep 2014 07:29:43 -0400
Subject: [alsa] hda/realtek: Change model name alias for ChromeOS

Message-id: <140972823573-258-patchset-jkysela@redhat.com>
Patchwork-id: 90958
O-Subject: [RHEL 7.1 PATCH 257/442] ALSA: hda/realtek - Change model name alias for ChromeOS
Bugzilla: 1112200
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>

Chrome OS was use model name of alc283-dac-wcaps for loading model as default.
Change the model name to same as model name of Chrome OS for future support.

Signed-off-by: Kailang Yang <kailang@realtek.com>
Signed-off-by: Takashi Iwai <tiwai@suse.de>

(picked from kernel: be8ef16a2508d12f4e2faa0933d1ffd02bf49f60)
rhbz#1112200
Signed-off-by: Jaroslav Kysela <jkysela@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/sound/pci/hda/patch_realtek.c b/sound/pci/hda/patch_realtek.c
index b6ec268..35a3c95 100644
--- a/sound/pci/hda/patch_realtek.c
+++ b/sound/pci/hda/patch_realtek.c
@@ -4782,7 +4782,7 @@ static const struct hda_model_fixup alc269_fixup_models[] = {
  {.id = ALC269_FIXUP_HP_GPIO_LED, .name = "hp-gpio-led"},
  {.id = ALC269_FIXUP_DELL1_MIC_NO_PRESENCE, .name = "dell-headset-multi"},
  {.id = ALC269_FIXUP_DELL2_MIC_NO_PRESENCE, .name = "dell-headset-dock"},
- {.id = ALC283_FIXUP_CHROME_BOOK, .name = "alc283-chrome"},
+ {.id = ALC283_FIXUP_CHROME_BOOK, .name = "alc283-dac-wcaps"},
  {.id = ALC283_FIXUP_SENSE_COMBO_JACK, .name = "alc283-sense-combo"},
  {}
 };
-- 
1.7.1